test/helpers/events.js

```diff
@@ -0,0 +1,15 @@
+export function dispatch (target, type, eventProperties) {
+	const event = new Event(type, { bubbles: true });
+	for (const property in eventProperties) {
+		event[property] = eventProperties[property];
+	} 
```

Comment:
```suggestion
	}
```
need to remove the trailing space to appease the linting rules